Fixes 
FAIL: g++.jason/thunk3.C (test for excess errors)
on sh4-unknown-linux-gnu.

Does any sh variant *not* use the generic thunk support?  If so, the patch should be
more cautious.  As it is, it excludes all sh processors from this test.

Index: thunk3.C
===================================================================
RCS file: /cvsroot/gcc/gcc/gcc/testsuite/g++.old-deja/g++.jason/thunk3.C,v
retrieving revision 1.13
diff -u -d -u -r1.13 thunk3.C
--- gcc-ss-3_3-20030714/gcc/testsuite/g++.old-deja/g++.jason/thunk3.C.old	25 Mar 2002 17:57:03 -0000	1.13
+++ gcc-ss-3_3-20030714/gcc/testsuite/g++.old-deja/g++.jason/thunk3.C	20 Jul 2003 16:57:38 -0000
@@ -2,7 +2,7 @@
 // Note that this will break on any target that uses the generic thunk
 //  support, because it doesn't support variadic functions.
 
-// excess errors test - XFAIL mips*-*-* rs6000-*-* powerpc-*-eabi m68k-*-coff m68k-motorola-sysv m88k-motorola-sysv3  mn10300-*-* mn10200-*-* v850-*-* sh-*-* sh64-*-* h8*-*-* xtensa-*-*
+// excess errors test - XFAIL mips*-*-* rs6000-*-* powerpc-*-eabi m68k-*-coff m68k-motorola-sysv m88k-motorola-sysv3  mn10300-*-* mn10200-*-* v850-*-* sh*-*-* h8*-*-* xtensa-*-*
 
 #include <stdarg.h>
 
